<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="main">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>425</x>
          <y>150</y>
          <width>131</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Bem Vindo</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="inserirEquipa">
       <widget class="QLabel" name="insNomeLabel">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>23</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nome da Equipa</string>
        </property>
       </widget>
       <widget class="QLabel" name="insEle1Label">
        <property name="geometry">
         <rect>
          <x>69</x>
          <y>93</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Elemento 1</string>
        </property>
       </widget>
       <widget class="QLabel" name="insEle2Label">
        <property name="geometry">
         <rect>
          <x>69</x>
          <y>163</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Elemento 2</string>
        </property>
       </widget>
       <widget class="QLabel" name="insEle3Label">
        <property name="geometry">
         <rect>
          <x>69</x>
          <y>233</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Elemento 3</string>
        </property>
       </widget>
       <widget class="QLabel" name="insEle4Label">
        <property name="geometry">
         <rect>
          <x>69</x>
          <y>303</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Elemento 4</string>
        </property>
       </widget>
       <widget class="QLabel" name="insEle5Label">
        <property name="geometry">
         <rect>
          <x>509</x>
          <y>93</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Elemento 5</string>
        </property>
       </widget>
       <widget class="QLabel" name="insEle6Label">
        <property name="geometry">
         <rect>
          <x>509</x>
          <y>163</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Elemento 6</string>
        </property>
       </widget>
       <widget class="QLabel" name="insEle7Label">
        <property name="geometry">
         <rect>
          <x>509</x>
          <y>233</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Elemento 7</string>
        </property>
       </widget>
       <widget class="QLabel" name="insEle8Label">
        <property name="geometry">
         <rect>
          <x>509</x>
          <y>303</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Elemento 8</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="insNomeLineE">
        <property name="geometry">
         <rect>
          <x>401</x>
          <y>20</y>
          <width>191</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="insEle1LineE">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>90</y>
          <width>191</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="insEle2LineE">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>160</y>
          <width>191</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="insEle3LineE">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>230</y>
          <width>191</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="insEle4LineE">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>300</y>
          <width>191</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="insEle5LineE">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>90</y>
          <width>191</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="insEle6LineE">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>160</y>
          <width>191</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="insEle7LineE">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>230</y>
          <width>191</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="insEle8LineE">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>300</y>
          <width>191</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="pushInserirEquipa">
        <property name="geometry">
         <rect>
          <x>385</x>
          <y>360</y>
          <width>110</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Inserir Equipa</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="inserirProva">
       <widget class="QLabel" name="insLocalLabel">
        <property name="geometry">
         <rect>
          <x>259</x>
          <y>140</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Local da Prova</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="insNomeProvaLineE">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>57</y>
          <width>241</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="insRobotLabel">
        <property name="geometry">
         <rect>
          <x>259</x>
          <y>220</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nome do Robot</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="insLocalLineE">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>137</y>
          <width>241</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="insNomeProvaLabel">
        <property name="geometry">
         <rect>
          <x>259</x>
          <y>60</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nome da Prova</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="insRobotLineE">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>217</y>
          <width>241</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="pushInserirProva">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>310</y>
          <width>110</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Inserir</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="inserirRobot">
       <widget class="QLabel" name="insNomeRobotLabel">
        <property name="geometry">
         <rect>
          <x>259</x>
          <y>93</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nome do Robot</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="insNomeRobotLineE">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>90</y>
          <width>241</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="insNomeEquipaLabel">
        <property name="geometry">
         <rect>
          <x>259</x>
          <y>180</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nome da Equipa</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="insNomeEquipaLineE">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>177</y>
          <width>241</width>
          <height>26</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="pushInserirRobot">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>290</y>
          <width>110</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Inserir</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="listarEquipa"/>
      <widget class="QWidget" name="listarProva"/>
      <widget class="QWidget" name="listarRobot"/>
      <widget class="QWidget" name="modificarEquipa"/>
      <widget class="QWidget" name="modificarProva"/>
      <widget class="QWidget" name="modificarRobot"/>
      <widget class="QWidget" name="eliminarEquipa"/>
      <widget class="QWidget" name="eliminarProva"/>
      <widget class="QWidget" name="eliminarRobot"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInserir">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>100</y>
      <width>135</width>
      <height>143</height>
     </rect>
    </property>
    <property name="title">
     <string>Inserir</string>
    </property>
    <addaction name="inserirEquipaAction"/>
    <addaction name="separator"/>
    <addaction name="inserirProvaAction"/>
    <addaction name="separator"/>
    <addaction name="inserirRobotAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuListar">
    <property name="geometry">
     <rect>
      <x>387</x>
      <y>100</y>
      <width>135</width>
      <height>143</height>
     </rect>
    </property>
    <property name="title">
     <string>Listar</string>
    </property>
    <addaction name="listarEquipaAction"/>
    <addaction name="separator"/>
    <addaction name="listarProvaAction"/>
    <addaction name="separator"/>
    <addaction name="listarRobotAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuModificar">
    <property name="title">
     <string>Modificar</string>
    </property>
    <addaction name="modificarEquipaAction"/>
    <addaction name="separator"/>
    <addaction name="modificarProvaAction"/>
    <addaction name="separator"/>
    <addaction name="modificarRobotAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEliminar">
    <property name="title">
     <string>Eliminar</string>
    </property>
    <addaction name="eliminarEquipaAction"/>
    <addaction name="separator"/>
    <addaction name="eliminarProvaAction"/>
    <addaction name="separator"/>
    <addaction name="eliminarRobotAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuModo_de_Prova">
    <property name="title">
     <string>Modo de Prova</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWelcome">
    <property name="title">
     <string>Welcome</string>
    </property>
   </widget>
   <addaction name="menuWelcome"/>
   <addaction name="menuInserir"/>
   <addaction name="menuListar"/>
   <addaction name="menuModificar"/>
   <addaction name="menuEliminar"/>
   <addaction name="menuModo_de_Prova"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="inserirEquipaAction">
   <property name="text">
    <string>Equipa</string>
   </property>
   <property name="toolTip">
    <string>Inserir uma Equipa</string>
   </property>
  </action>
  <action name="inserirProvaAction">
   <property name="text">
    <string>Prova</string>
   </property>
   <property name="toolTip">
    <string>Inserir uma Prova</string>
   </property>
  </action>
  <action name="inserirRobotAction">
   <property name="text">
    <string>Robot</string>
   </property>
   <property name="toolTip">
    <string>Inserir um Robot</string>
   </property>
  </action>
  <action name="listarEquipaAction">
   <property name="text">
    <string>Equipa</string>
   </property>
   <property name="toolTip">
    <string>Listar uma Equipa</string>
   </property>
  </action>
  <action name="listarProvaAction">
   <property name="text">
    <string>Prova</string>
   </property>
   <property name="toolTip">
    <string>Listar uma Prova</string>
   </property>
  </action>
  <action name="listarRobotAction">
   <property name="text">
    <string>Robot</string>
   </property>
   <property name="toolTip">
    <string>Listar um Robot</string>
   </property>
  </action>
  <action name="modificarEquipaAction">
   <property name="text">
    <string>Equipa</string>
   </property>
   <property name="toolTip">
    <string>Modificar uma Equipa</string>
   </property>
  </action>
  <action name="modificarProvaAction">
   <property name="text">
    <string>Prova</string>
   </property>
   <property name="toolTip">
    <string>Modificar uma Prova</string>
   </property>
  </action>
  <action name="modificarRobotAction">
   <property name="text">
    <string>Robot</string>
   </property>
   <property name="toolTip">
    <string>Modificar um Robot</string>
   </property>
  </action>
  <action name="eliminarEquipaAction">
   <property name="text">
    <string>Equipa</string>
   </property>
   <property name="toolTip">
    <string>Eliminar uma Equipa</string>
   </property>
  </action>
  <action name="eliminarProvaAction">
   <property name="text">
    <string>Prova</string>
   </property>
   <property name="toolTip">
    <string>Eliminar uma Prova</string>
   </property>
  </action>
  <action name="eliminarRobotAction">
   <property name="text">
    <string>Robot</string>
   </property>
   <property name="toolTip">
    <string>Eliminar uma Robot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
